Description: Round matrix entry in GraphicalModelsMLE test; was
 failing on i386, arm64, ppc64el, and s390x.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1685
Origin: https://github.com/Macaulay2/M2/pull/1687
Last-Update: 2020-12-12

--- a/M2/Macaulay2/packages/GraphicalModelsMLE.m2
+++ b/M2/Macaulay2/packages/GraphicalModelsMLE.m2
@@ -1790,7 +1790,7 @@
 (mx,MLE,ML)= solverMLE (G,U,ChooseSolver=>"NAG4M2")
 assert(round(5,mx)==-8.4691)
 assert(ML==1)
-assert(MLE_(2,0)==0)
+assert(round(6,MLE_(2,0))==0)
 assert(round(6,MLE_(1,1))== 1.1875)
 assert(round(6,MLE_(3,2))==   3.264929)
 ///
